app/views/layouts/Layout.scala.html

```diff
@@ -75,7 +80,7 @@
 
 @govukLayout(
     pageTitle = Some(messages("common.pageTitle", pageTitle, "Appeal a VAT penalty")),
-    headBlock = Some(headBlock),
+    headBlock = Some(hmrcHead(nonce = nonce, headBlock = additionalHeadBlock)),
     headerBlock = Some(header()),
     scriptsBlock = Some(scripts),
```

Comment:
@ruthdas03 I think this is what you need to get it to work
```suggestion
    scriptsBlock = Some(hmrcScripts(nonce=nonce, scriptsBlock=scripts)),
```